code/_onclick/hud/blob_overmind.dm

```diff
@@ -53,8 +53,13 @@
 
 /atom/movable/screen/blob/blobbernaut
 	icon_state = "ui_blobbernaut"
-	name = "Produce Blobbernaut (40)"
-	desc = "Produces a strong, smart blobbernaut from a factory blob for 40 resources.<br>The factory blob used will become fragile and unable to produce spores."
+	name = "Produce Blobbernaut (ERROR)"
+	desc = "Produces a strong, smart blobbernaut from a factory blob for (ERROR) resources.<br>The factory blob used will become fragile and unable to produce spores."
+
+/atom/movable/screen/blob/blobbernaut/Initialize()
+	..()
```

Comment:
```suggestion
	. = ..()
```
Always return the parent call value.
On `Initialize()` this is a bit more critical.